static/panes/pane.ts

```diff
@@ -92,6 +87,15 @@ export abstract class Pane<S> {
         this.registerOpeningAnalyticsEvent();
     }
 
+    protected initializeCompilerInfo(state: Record<string, any>) {
+        this.compilerInfo = {
+            compilerId: state.id,
+            compilerName: state.compilerName,
+            editorId: state.editorid,
+            treeId: state.treeid,
```

Comment:
Is anything needed in `override initializeCompilerInfo` for `treeId` vs `treeid`?

Reply:
good question, not sure, will look into it

Reply:
i added them, though don't think it matters much because they can be restored from the compiler